flow/designs/src/chameleon_hier/rtl/ibex/ibex_core.nangate.out.sdc

```diff
@@ -0,0 +1,35 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set_driving_cell [all_inputs] -lib_cell BUF_X2
+set_load 10.0 [all_outputs]
+
+#============ Auto-generated from config ============
+create_clock -name clk_i -period 4.0 {clk_i}
+set_output_delay -clock clk_i 1.2000000000000002 instr_req_o
+set_output_delay -clock clk_i 1.2000000000000002 instr_addr_o
+set_output_delay -clock clk_i 1.2000000000000002 data_req_o
+set_output_delay -clock clk_i 1.2000000000000002 data_we_o
+set_output_delay -clock clk_i 1.2000000000000002 data_be_o
+set_output_delay -clock clk_i 1.2000000000000002 data_addr_o
+set_output_delay -clock clk_i 1.2000000000000002 data_wdata_o
+set_output_delay -clock clk_i 0.7999999999999998 core_sleep_o
```

Comment:
No reason to have clock periods with floating point errors as the input.

Reply:
Also SDC files should not be in the `designs/src` folder they should be in `designs/[platform]`

Reply:
Done, SDC is now defined only under designs/[platform[.